### PR TITLE
schema: marc21 tags

### LINKED ISSUE #35: RFC: Adding marc tags to field descriptions
Currently, the schema for Marc21 contain the "human readable marc tag". E.g. ""title_statement" gives

```
"description": "Title Statement"
```

It would be helpful if we could also hold the numeric tag, ie. `245`. 
- This serves as sort of a unique identifier
- It helps inline documentation
- It is easy to construct a lookup url form that (e.g. http://www.loc.gov/marc/bibliographic/bd245.html), that can serve well in a help functionality.
- It might serve in some upcoming editor for a "non-human" format (being usually preferred by full time cataloguers due to it's compactness).

Without changing the model one could just add that to "description". Similarly, it might be very helpul to have the subfield codes in some sort of description as well (due to their uniqueness). E.g.

```
"title_statement": {
        "description": "Title Statement",
        "tag": "245",
        "title": {
             "type": "string",
             "subfield" : "a"
        },
```

### PR DESCRIPTION
**Pull request by SamiHiltunen**
- Adds marc21 tags to schemas. (closes #35)

Signed-off-by: Sami Hiltunen sami.mikael.hiltunen@cern.ch

### CODE REVIEW

File: dojson/utils.py [hunk 1]
```diff
@@ -104,6 +105,18 @@ def dump(iterator):
     return json.dumps(list(iterator))
 
 
+def deprecated(datafield, explanation):
+    """Decorate as deprecated."""
+    def decorator(f):
+        @functools.wraps(f)
+        def wrapper(self, key, values, **kwargs):
+            warnings.warn('{0}: {1}'.format('datafield', 'explanation'),
+                          warnings.DeprecationWarning)
```
Comment (jirikuncar) [inline dojson/utils.py]: IMHO you can use just `DeprecationWarning`

**Comment by jirikuncar** [comment]
@SamiHiltunen can you separate `dojson: *` commits to new PR? We need to think about extending JSON schema with custom fields.

**Comment by SamiHiltunen** [comment]
@jirikuncar separated them here --> #127

**Comment by jirikuncar** [comment]
I'm closing the PR as we might generate JSON form schema instead.

**Pull request closed without merging**